### ISSUE TITLE
Suggested rule: native_function_invocation

### ISSUE THREAD

**Issue by alexmanno** [facile-it/facile-coding-standard#10]
Hi guyz, 
I believe we should add this rule to improve performance:
```
'native_function_invocation' => true
```

This rule adds a slash before the native functions. Example:

Before:
```php
$str = json_encode($var, true);
```

After:
```php
$str = \json_encode($var, true);
```

**Comment by Jean85** [comment]
I agree with this :+1: 
Obviously this would mean a 0.3 release. Do we have any other new rule that we would like to add in the same BC release?

**Comment by thomasvargiu** [comment]
Implemented in #16
